### PR TITLE
Added markdown feature to card description

### PR DESCRIPTION
**Pull request by iraycd**

### CODE REVIEW

File: client/views/cards/templates.html [hunk 1]
```diff
@@ -181,7 +181,9 @@ <h3 class="card-detail-item-header">Labels</h3>
                             <h3 class="card-detail-item-header js-show-with-desc">Description</h3>
                             <a href="#" class="card-detail-item-header-edit hide-on-edit js-show-with-desc js-edit-desc {{ isMemberAll '' 'hide' }}">Edit</a>
                             <div class="current markeddown hide-on-edit js-card-desc js-show-with-desc">
-                                <p>{{ card.description }}</p>
+                              {{#markdown}}
+{{ card.description }}
```
Comment (mquandalle) [inline client/views/cards/templates.html]: I would write it in one single line `{{# markdown }}{{ card.description }}{{/ markdown }}` inside the `<p>` tag.

Comment (iraycd) [inline client/views/cards/templates.html]: :+1:

**Comment by mquandalle** [comment]
We probably also want markdown support for card comments

**Comment by mquandalle** [comment]
Thanks, rebased on master 657a8f9a0c98c96776497e784a8abf7da54dc2f0

**Pull request closed without merging**